### PR TITLE
Add new button styles

### PR DESCRIPTION
**Pull request by omarryhan**

### CODE REVIEW

File: packages/components/theme.ts
Comment (achrafismyname) [inline packages/components/theme.ts:20]: And it would be nice if you add a simple hover effect. Just like lower the opacity on hover or anything that show we are hovering the button

File: packages/components/theme.ts
Comment (achrafismyname) [inline packages/components/theme.ts:10]: The button text has currently no styles, use variants to style them. Just add like ``variant: 'text.buttons.0'`` with the variant you think it match best

You can find all variants inside theme.ts in the `wip_design_update` branch
https://github.com/curaOS/source/blob/af9c891fc8c8b64622de273081fabf844cca7e72/packages/components/theme.ts#L38-L74

File: packages/components/theme.ts
Comment (achrafismyname) [inline packages/components/theme.ts:26]: You can use `pt` instead of `paddingTop` and `pb` instead of `paddingBottom` and `py` to define top and bottom paddign at the same time

**Review by achrafismyname — CHANGES_REQUESTED** [review]
Thanks for your submit! I just have some simple comments before we can merge this.

We were already working on [theme.ts](https://github.com/curaOS/source/blob/wip_design_update/packages/components/theme.ts) from the `wip_design_update` branch and it's ahead of the `master` branch that you worked on, so we can't merge this unless you copy this changes to `wip_design_update` instead of `master`.

**Comment by omarryhan** [comment]
Hey, I resolved all the issues in a new PR #119. I'll go ahead and close this one now

**Pull request closed without merging**
